### PR TITLE
Cortx 29708: Handle validations for all the arguments and handlers in mock_health_event_publisher.py file.

### PR DESCRIPTION
**Pull request by gargpalak048**
# Problem Statement
- After current implementation, some end case scenarios (i.e for handlers like publish, get-disks etc) were not properly handled.

# Design
- Updated the list of arguments to validate.

# Coding
-  [x] Coding conventions are followed and code is consistent

# Testing 
- [x] Unit and System Tests are added
https://jts.seagate.com/secure/attachment/514444/test_results_29708.txt
- [x] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Review Checklist 
- [x] PR is self reviewed
- [x] JIRA number/GitHub Issue added to PR
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained
- [ ] Is there a change in filename/package/module or signature? [Y/N]: 
- [ ] If yes for above point, is a notification sent to all other cortx components? [Y/N]
- [ ] Side effects on other features (deployment/upgrade)? [Y/N]
- [ ] Dependencies on other component(s)? [Y/N]
-     If yes for above point, post link to the corresponding PR.

# Review Checklist 
- [ ] Is perfline test run and the report with and without the changes updated in the PR? [Y/N]: 

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by shaileshsvc — APPROVED** [review]
Approved

**Review by akash2144 — APPROVED** [review]

**Comment by ArchanaLimaye** [comment]
@gargpalak048  please confirm that -h option also displays correct help

File: ha/util/health_generator/mock_health_event_publisher.py [hunk 1]
```diff
@@ -206,14 +206,16 @@ def get_args() -> (argparse.Namespace, argparse.ArgumentParser):
...
+    options_regex = re.compile('(-gdt$)|(-gs$)|(--get-data-nodes$)|(--get-server-nodes$)\
+                               |(publish$)|(get-disks$)|(get-cvgs$)')
 
     if len(sys.argv) > 1:
         option = sys.argv[1]
         if not options_regex.match(option):
             print(f"Error: Invalid Argument: {option}\n"
                   f"Please check supported args list:"
-                  "['-gdt|--get-data-nodes', '-gs|--get-server-nodes']")
+                  "['-gdt|--get-data-nodes', '-gs|--get-server-nodes']\n"
```
Comment (Madhura-08) [inline ha/util/health_generator/mock_health_event_publisher.py:217]: Can we call parser_obj.print_help() here instead?

Comment (gargpalak048) [reply ha/util/health_generator/mock_health_event_publisher.py]: Agree, Incorporated

**Review by gauri-bhosale — APPROVED** [review]

**Review by Madhura-08 — APPROVED** [review]

File: ha/util/health_generator/mock_health_event_publisher.py (hunk 1 above)
Comment (Madhura-08) [reply ha/util/health_generator/mock_health_event_publisher.py]: Thanks!

**Review by mukhtar-inamdar — COMMENTED** [review]
approved

**Pull request merged**